features/docs.feature

```diff
@@ -0,0 +1,189 @@
+          type: git
+          url: https://github.com/partcad/partcad-index.git
+      EOF
+      """
+    Then the command should exit with a status code of "0"
+    When I run "pc list"
+    Then the command should exit with a status code of "0"
+    When I run "echo "translate (v= [0,0,0])  cube (size = 10);" > test.scad"
+    Then the command should exit with a status code of "0"
+    When I run "pc add part scad test.scad"
+    Then the command should exit with a status code of "0"
+    When I run "pc inspect :test"
+    Then the command should exit with a status code of "0"
+    When I run "pc render -t stl :test"
+    Then the command should exit with a status code of "0"
+
+
+  @docs-design
+  Scenario: Design: Parametrized objects
+    When I run "pc init"
+    Then the command should exit with a status code of "0"
+    When I run command:
+      """
+      pc \
+        inspect \
+          -p length=30 \
+          -p size=M4-0.7 \
+          /pub/std/metric/cqwarehouse:fastener/hexhead-din931
+      """
+    # TODO: Failure to pip install from GH repo
+    Then the command should exit with a status code of "1"
+    When I run command:
+      """
+      pc \
+        inspect \
+          /pub/std/metric/cqwarehouse:fastener/hexhead-din931;length=30,size=M4-0.7
+      """
+    Then the command should exit with a status code of "0"
+     And STDOUT should contain 'DONE: InitCtx:'
+     And STDOUT should contain 'Visualizing in "OCP CAD Viewer"...'
+     And STDOUT should contain 'Unable to open config file /tmp/sandbox/home/partcad-cli-'
+     And STDOUT should contain 'No VS Code or "OCP CAD Viewer" extension detected.'
+     And STDOUT should contain 'DONE: inspect: this'
+
+   @docs-design
+  Scenario: Design: Objects in a cart
+    When I run "pc init"
+    Then the command should exit with a status code of "0"
+    When I run command:
+      """
+      pc supply quote \
+        --provider /pub/svc/commerce/gobilda:gobilda \
+        /pub/robotics/multimodal/openvmp/robots/don1:assembly-wormgear#10
+      """
+    # TODO: We don't have enough XXX stock on hand for the quantity you selected
+    Then the command should exit with a status code of "1"
+     And STDOUT should contain 'DONE: SupplyQuote: this:'
+     And STDOUT should contain 'The following quotes are received:'
+     And STDOUT should contain '/pub/svc/commerce/gobilda:gobilda: No quote received'
+
+  # TODO: This depends on ~/.partcad/config.yaml:googleApiKey
+  @wip @docs-features
+  Scenario: Features: Design
+    When I run command:
+      """
+      mkdir -pv $HOME/.partcad
+      cat <<EOF > $HOME/.partcad/config.yaml
+      # https://aistudio.google.com/apikey
+      # TODO: Create dedicated token for CI and create version of test for success scenario
+      googleApiKey: $PARTCAD_GOOGLE_API_KEY_MISSING
+      # YOLO settings to speed up the process
+      maxGeometricModeling: 1
+      maxModelGeneration: 1
+      maxScriptCorrection: 1
+      EOF
+      """
```

Comment:
I'm still not sure if it's okay to use live 3rd parties services during PR checks but from other had if it does not work in PRs for any reason same will happen for end user, so IMHO best part here is to have dedicated GitHub Actions token with spending limit.